submissions/templates/submissions/success.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+    <h1>YAY</h1>

```

Comment:
Maybe flesh out the success message a little and add a link back to some other page.